README.md

```diff
@@ -60,46 +60,187 @@ xcode-select --install
 
 ## Users Guide
 
-To use the plugin and invoke it as a pipeline step, follow these instructions:
-
-### Synopsys Security Scan - Black Duck
+### Bitbucket Prerequisites:
+
+#### Bitbucket token for your Job Configuration:
+Account-Level Bitbucket HTTP Access Token is required to configure your Job. <bR>
+To Generate this token, follow these instructions:
+>- Select Profile Photo → Manage Account → HTTP Access Tokens → Create token
+>- Enter Token name
+>- Keep everything as default or you can change the Project/Repository Permissions as your need.
+>- Click the Create Button. Then a token will be generated. <br>
+** You need to store this token to configure the Branch Sources of your Jenkins job
+
+#### Bitbucket token for PrComment/FixPr:
+bitbucket_token parameter is required as input when running Black Duck/Coverity PR Comment. 
+There are two different types of tokens in bitbucket which can be passed to bitbucket_token
+parameter.
+
+**1. Account-Level Bitbucket HTTP Access Token:**   
+To use this token for PR comments, it must hold Project permissions such as "Project write" or "Project admin." This token is employed when working both on the Project level and repository level.
+We described how can we get this token on the upper section.
+
+**2. Repository-Level Bitbucket HTTP Access Token:**   
+To use this token for PR comments,
+it must hold Repository permissions such as "Repository write" or "Repository admin."
+This token is employed when working at the repository level. To Generate this token, follow these instructions:
+>- First go to the source of your repository.
+>- Click on the repostory settings icon.
+>- Then click on the HTTP access tokens.
+>- Next click Create token button.
+>- Enter Token name.
+>- Keep everything as default or you can change the Project/Repository Permissions as your need.
+>- Click the Create Button. Then a token will be generated. <br>
+** You need to store this token to run the Black Duck/Coverity PR Comment Feature.
+
+### Project Setup
+#### Installing Helper Plugins for Jenkins:
+1. Bitbucket Branch Source 
```

Comment:
Please check if user needs to install Bitbucket Branch source plugin, as we already have this dependency in  `build.gradle`.